### ISSUE TITLE
MultiMAE: Multi-modal Multi-task Masked Autoencoders

### ISSUE THREAD

**Issue by dalearnz0ne** [dalearnz0ne/COOL_PROJECTS#1]
## [MultiMAE: Multi-modal Multi-task Masked Autoencoders](https://multimae.epfl.ch/)
We introduce Multi-modal Multi-task Masked Autoencoders (MultiMAE), an efficient and effective pre-training strategy for Vision Transformers. Given a small random sample of visible patches from multiple modalities, the MultiMAE pre-training objective is to reconstruct the masked-out regions.

![](https://storage.googleapis.com/multimae_bucket/images/pretrain_figure.svg)
